usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -19,6 +20,104 @@
 ========
 """
 
+"""@pylatest api/gluster.cluster_create
+API-gluster: cluster_create
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Positive create gluster cluster.
+"""
+
+
+def test_cluster_create_valid(
+        valid_session_credentials,
+        valid_nodes,
+        net_interface="eth0"):
+    """@pylatest api/gluster.cluster_create
+        .. test_step:: 1
+
+        Check if there is at least one gluster node for cluster creation.
+
+        .. test_result:: 1
+
+        Test passes if there is at least one gluster node.
+        """
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    pytest.check(
+        len(valid_nodes) > 0,
+        "There have to be at least one gluster node."
+        "There are {}".format(len(valid_nodes)))
+    """@pylatest api/gluster.cluster_import
+        .. test_step:: 2
+
+            Send POST request to Tendrl API ``APIURL/GlusterCreateCluster
+
+        .. test_result:: 2
+
+            Server should return response in JSON format:
+
+                {
+                  "job_id": job_id
+                }
+
+            Return code should be **202**
+                with data ``{"message": "Accepted"}``.
+
+        """
+    nodes = []
+    provisioner_ip = None
+    network = None
+    for x in valid_nodes:
+        if "tendrl/server" in x["tags"]:
+            network = "{}/22".format(
+                ast.literal_eval(x["networks"][net_interface]["ipv4"])[0])
```

Comment:
Tendrl returns string in format "[\"some.ip.address\"]". I just want the string of valid ip. If you know how to make it more elegantly then be my guest. I will gladly correct it. :)